app/assets/javascripts/application.js.erb

```diff
@@ -176,8 +177,9 @@ function display_node_rows( d ) {
 function display_enhanced_job_data( d ) {
 
     if (is_defined(d.error)) {
-        return      '<div><tr><td>'+ d.error +'</td></tr></div>';
+        return      '<div><tr><td><div class="alert alert-warning">No job details because job has already left the queue. </div></td></tr></div>';
```

Comment:
Also, this message appears in the view only after 
1) clicking on an existing job
2) getting a response from the server
In the case of a 500 error, this message would not be displayed.